osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -112,5 +117,62 @@ private void load(TextureStore textures)
                 Sprite.Texture = Beatmap?.Background ?? textures.Get(@"Backgrounds/bg1");
             }
         }
+
+        public class DimmableBackground : UserDimContainer
+        {
+            /// <summary>
+            /// The amount of blur to be applied to the background in addition to user-specified blur.
+            /// </summary>
+            /// <remarks>
+            /// Used in contexts where there can potentially be both user and screen-specified blurring occuring at the same time, such as in <see cref="PlayerLoader"/>
+            /// </remarks>
+            public readonly Bindable<float> BlurAmount = new Bindable<float>();
+
+            public Background Background
+            {
+                get => background;
+                set
+                {
+                    base.Add(background = value);
```

Comment:
Should we first remove/expire any existing `Background`?  We certainly don't want to be adding the same `Background` multiple times either.

Reply:
Not a change from this PR, but definitely dangerous. Will address.

Reply:
If `Background` is a special case, should it be done via an internal child instead?

Reply:
This will fail if you reassign the same background, as it will try to add an expired `Drawable`.

Reply:
which can never happen. check how this class is used